catalogue/aarch64-ifetch/tests/DIC0-IDC0/MP.RF+dc-dsb-ic-dsb+addr-1.litmus

```diff
@@ -0,0 +1,16 @@
+AArch64 MP.RF+dc-dsb-ic-dsb+addr-1
```

Comment:
This test is allowed intentionally.

It might be a misnomer to say `addr` in the name of the test, as herd7 / aarch64deps.cat do not define this to be an address dependency. We could make the cat file draw such edges for illustration purposes, but not define them as required ordering.